plugin-tester/src/main/java9/com/example/Example.java

```diff
@@ -0,0 +1,13 @@
+//package com.example;
+//
+//import scala.Console;
+//
+//public class Example {
```

Comment:
If we just have both added as source directories, then javac will fail since the same class was defined in two places